src/frontend/komponenter/Behandling/Vurdering/VurderingUtil.ts

```diff
@@ -23,7 +36,11 @@ export const harBesvartPåAlleDelvilkår = (delvilkårsvurderinger: IDelvilkår[
 export const skalViseLagreKnapp = (vurdering: IVurdering, config: IVilkårConfig): boolean => {
     const { begrunnelse, delvilkårsvurderinger } = vurdering;
 
-    if (manglerBegrunnelse(begrunnelse)) {
+    const begrunnlsePåkrevd =
+        config.begrunnelsePåkrevdHvisOppfylt === undefined
```

Comment:
Er dette riktig tolkning: at undefined og true krever begrunnelse, men false gjør at man ikke krever? 

Kan vi isåfall endre config til å "ikke ha" undefined - og sette true på alle som skal ha?

Reply:
Evt snu på det: trengerIkkeBegrunnelseHvisOppfylt ?

Reply:
Ja, du tolket det riktig. Enig i forslaget ditt 👍